sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/jdbc/JDBCOptions.scala

```diff
@@ -17,47 +17,130 @@
 
 package org.apache.spark.sql.execution.datasources.jdbc
 
+import java.sql.{Connection, DriverManager}
+import java.util.Properties
+
 /**
  * Options for the JDBC data source.
  */
 class JDBCOptions(
     @transient private val parameters: Map[String, String])
   extends Serializable {
 
+  import JDBCOptions._
+
+  def this(url: String, table: String, parameters: Map[String, String]) = {
+    this(parameters ++ Map("url" -> url, "dbtable" -> table))
+  }
+
+  val asProperties: Properties = {
```

Comment:
Sounds good but it seems both are fine as actual class name is `Properties` but I guess the meaning is connection properties. Maybe this one depends on personal taste.